apps/coscrad-frontend/src/components/resources/bibliographic-references/bibliographic-reference-detail.container.tsx

```diff
@@ -1,10 +1,5 @@
-import { useLoadableBibliographicReferenceById } from '../../../store/slices/resources';
-import { AggregateDetailContainer } from '../../higher-order-components/aggregate-detail-container';
-import { BibliographicReferenceDetailPresenter } from './bibliographic-reference-detail.presenter';
+import { ResourceType } from '@coscrad/api-interfaces';
+import { AggregatePage } from '../../higher-order-components/aggregate-page';
 
-export const BibliographicReferenceDetailContainer = (): JSX.Element => (
-    <AggregateDetailContainer
-        useLoadableSearchResult={useLoadableBibliographicReferenceById}
-        DetailPresenter={BibliographicReferenceDetailPresenter}
-    />
-);
+export const BibliographicReferenceDetailContainer = (): JSX.Element =>
+    AggregatePage(ResourceType.bibliographicReference);
```

Comment:
Is the Aggregate Page actually just a Resource Page, or can we use it for the other non-resource Aggregates as well?  Based on past experience with the back-end, we should stop and answer this question right now. It's wasted effort to rewrite it to be less general or more general later.

Reply:
Let's make it a ResourcePage for now but leave a comment.